docs/reference/configuration.rst

```diff
@@ -291,6 +299,22 @@ multiple paragraphs, if necessary. The long description *must not* be a copy of
 the ``description``, or include the ``description`` as the first line of the
 ``long_description``.
 
+``primary_color``
+~~~~~~~~~~~~~~~~~
+
+A hexadecimal RGB color value (e.g., ``#008577``) to use as the primary color
+for the application. On Android this will be used to color the app bar in the
+main window. This setting is only used if the platform allows color
+modification, otherwise it is ignored.
```

Comment:
Notes about the specific platform interpretations should be in the relevant platform guide, unless they're indicative of behavior on all platforms.
```suggestion
A hexadecimal RGB color value (e.g., ``#008577``) to use as the primary color
for the application. This setting is only used if the platform allows color
modification, otherwise it is ignored.
```